packages/settings-library/src/settings_library/tracing.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import AnyUrl, Field
+
+from .base import BaseCustomSettings
+
+
+class TracingSettings(BaseCustomSettings):
+    TRACING_ZIPKIN_ENDPOINT: AnyUrl = Field(
```

Comment:
all other settings capture ``*_entrypoint``s  by splitting in ``*_host`` and ``*_port``. 

Based on [jaeger](https://www.jaegertracing.io/docs/1.26/getting-started/#all-in-one) container exposed ports:
```.env
TRACING_ZIPKIN_HOST: srt = "jaeger"
TRACING_ZIPKIN_PORT: int = Field(9411, description="HTTP | collector | Zipkin compatible endpoint")
TRACING_ZIPKIN_THRIFT_COMPACT_PORT: int = Field(5775, description="UDP | collector| accept zipkin.thrift over compact thrift protocol (deprecated, used by legacy clients only)")
```

and then use the ``cached_property`` approach to compose urls. (see e.g. ``PostgreSettings``)


- Based on the documentation, 5775 seems deprecated by 




Port | Protocol | Component | Function
-- | -- | -- | --
5775 | UDP | agent | accept zipkin.thrift over compact thrift protocol (**deprecated**, used by legacy clients only)
6831 | UDP | agent | accept jaeger.thrift over compact thrift protocol
6832 | UDP | agent | accept jaeger.thrift over binary thrift protocol
5778 | HTTP | agent | serve configs
16686 | HTTP | query | serve frontend
14268 | HTTP | collector | accept jaeger.thrift directly from clients
14250 | HTTP | collector | accept model.proto
9411 | HTTP | collector | Zipkin compatible endpoint (optional)

<br class="Apple-interchange-newline">

Reply:
I thought we discussed this already and said that endpoints would be ok as well. ok will review it.